### ISSUE TITLE
Merge in BLE code to existing BT and USB Host code.

### ISSUE THREAD

**Issue by K7MDL2** [K7MDL2/IC-705-BLE-Serial-Example#14]
BLE is working in another test program on the CoreS3. Merge it in with compile time #defines and if active (as appropriate), switch between BLE, USBHost, and BT.  

For BT versions, CoreS3 does only BLE, Core Basic only BT Classic.  Core2 should do all 3, TBD.  I now have one of each model to test on.

**Comment by K7MDL2** [comment]
To finish this 
1. Merge TX polling mechanisms
2. Figure out how to receive unsolicited transmissions from the radio.  Right now I only see messages through the NotifyOnCallback mechanism which seems to only work in response to transmission (polling event).  Messages such as radio side VFO updates are received in a group behind each poll result.  Doing high speed polling for PTT help unblock those but there should be a way to see any waiting change.

Will open a separate issue for tuning the advertisement process.  I have observed some issues with that.

**Comment by K7MDL2** [comment]
Seems that just sending RX -> canNotify() at a fast clip will release them, no actual Rx-< write() required.  Modified SendMessage form String to use uint8_t, length, false and once the length was correct, it stated working.  PTT is fast enough to unloaded VFO and band changes (mode/filter+vfo).  Moving on.
